main_interactive.jl

```diff
@@ -0,0 +1,38 @@
+include("src/Diffusion.jl")
```

Comment:
I am not sure how the file `main_interactive` should be used but it might be that this should say

```suggestion
import Diffusion
```

Reply:
You are right this is better: it forces the user to activate the project Diffusion or else the module cannot be imported. This ensures that in the interactive mode the user will always use the same environment as used for building the shared library.